src/components/orders/StatusLabel/index.tsx

```diff
@@ -33,6 +41,18 @@ function setStatusColors({ theme, status }: { theme: DefaultTheme; status: Order
       text = theme.labelTextOpen
       background = theme.labelBgOpen
       break
+    case 'partially filled':
+      return css`
+        background: linear-gradient(270deg, rgba(0, 216, 151, 0.1) 0%, rgba(217, 109, 73, 0.1) 100%);
```

Comment:
I'm not sold on the gradient.
I rather keep it solid

Reply:
Done! 
![image](https://user-images.githubusercontent.com/4270166/183440448-b5da5526-7301-4c4c-8a34-8545ac20ad5e.png)

Reply:
Oh, I missed this gradient!
I actually saw the one Elena mentioned below and thought that was the only one.

Could you make the `Partially filled` status solid as well?